server/chat.ts

```diff
@@ -494,7 +494,7 @@ export class CommandContext extends MessageContext {
 
 		if (typeof commandHandler === 'function' && requireGlobalCommand) {
 			const baseCmd = typeof curCommands[cmd] === 'string' ? curCommands[cmd] : cmd;
-			if (!curCommands['!' + baseCmd]) {
+			if (!curCommands['!' + baseCmd] && !this.user.can('bypassall')) {
```

Comment:
This seems like the wrong line to do it in.

`"!cmdname": true,` is an opt-in sigil meaning "this command won't crash if you use it outside of a chatroom". If there's a specific command you want to use but can't, just add that sigil.